### ISSUE TITLE
AppVeyor pre-3.5 tests are failing

### ISSUE THREAD

**Issue by mattsb42-aws** [aws/aws-encryption-sdk-python#159]
Figure out why

ex:

https://ci.appveyor.com/project/mattsb42-aws/aws-encryption-sdk-python-qvyet/builds/25075033

**Comment by caitlin-tibbetts** [comment]
I'll look into this.

**Comment by mattsb42-aws** [comment]
Update: looks like now all pre-3.5 versions are failing, now for possibly different reasons... :(

ex:
https://ci.appveyor.com/project/mattsb42-aws/aws-encryption-sdk-python-qvyet/builds/26119032

**Comment by caitlin-tibbetts** [comment]
For Python34, the issue seems to be that the version of the module `ast` in the Python library that AppVeyor is running to test Python34 does not include the attribute `MatMult`.  This is confirmed in the docs for the version of `ast` run in [3.4](https://docs.python.org/3.4/library/ast.html).  According to the stack trace, this version of `ast` is used in `runpy`, which is a dependency for `pytest`.

```
Traceback (most recent call last):
  File "c:\python34-x64\Lib\runpy.py", line 170, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python34-x64\Lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\projects\aws-encryption-sdk-python-qvyet\.tox\py34-examples\Scripts\pytest.EXE\__main__.py", line 5, in <module>
  File "c:\projects\aws-encryption-sdk-python-qvyet\.tox\py34-examples\lib\site-packages\pytest.py", line 6, in <module>
    from _pytest.assertion import register_assert_rewrite
  File "c:\projects\aws-encryption-sdk-python-qvyet\.tox\py34-examples\lib\site-packages\_pytest\assertion\__init__.py", line 6, in <module>
    from _pytest.assertion import rewrite
  File "c:\projects\aws-encryption-sdk-python-qvyet\.tox\py34-examples\lib\site-packages\_pytest\assertion\rewrite.py", line 443, in <module>
    ast.MatMult: "@",
AttributeError: 'module' object has no attribute 'MatMult'
```

**Comment by caitlin-tibbetts** [comment]
The offending code is in `c:\projects\aws-encryption-sdk-python-qvyet\.tox\py34-examples\lib\site-packages\_pytest\assertion\rewrite.py`
```
# Python 3.5+ compatibility
try:
    binop_map[ast.MatMult] = "@"
except AttributeError:
    pass
```

**Comment by acioc** [comment]
Closed stale PR for this issue

**Comment by farleyb-amazon** [comment]
CI has been fixed, plus we no longer support <3.5. Resolving.
